src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -103,7 +103,7 @@ public void Initialize()
                 Trace.WriteLine(string.Format("    {0} = {1}", "ValidityPeriod", _validityPeriod));
                 Trace.WriteLine(string.Format("    {0} = {1}", "Valid to", _validityNotAfter));
 
-                _authorityCertificate = CreateCertificate(true, false, null, string.Empty);
+                _authorityCertificate = CreateCertificate(true, false, null, null);
```

Comment:
What's the parameter naming syntax I&Jason shall follow? Could you give an example?